recipes/karchive/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.5)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(Qt5 CONFIG REQUIRED Core)
+
+add_executable(example example.cpp)
+target_link_libraries(example CONAN_PKG::karchive Qt5::Core)
```

Missing final endline.
```suggestion
target_link_libraries(example CONAN_PKG::karchive Qt5::Core)
```
I don't know whether this suggestion can fix these kinds of error.